op-acceptor/config.go

```diff
@@ -35,26 +36,34 @@ func NewConfig(ctx *cli.Context, log log.Logger, testDir string, validatorConfig
 		return nil, errors.New("test directory is required")
 	}
 	if validatorConfig == "" {
-		return nil, errors.New("validator config path is required")
-	}
-	if gate == "" {
-		return nil, errors.New("gate is required")
+		return nil, errors.New("validator configuration file is required")
 	}
 
-	// Get absolute paths
+	// Resolve the absolute paths
 	absTestDir, err := filepath.Abs(testDir)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get absolute path for test dir: %w", err)
+		return nil, fmt.Errorf("failed to resolve absolute path for test directory: %w", err)
 	}
+
 	absValidatorConfig, err := filepath.Abs(validatorConfig)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get absolute path for validator config: %w", err)
+		return nil, fmt.Errorf("failed to resolve absolute path for validator config: %w", err)
```

Comment:
Same as above - smack the `validatorConfig` in the error message as well

Reply:
Done